omim2obo/parsers/omim_txt_parser.py

```diff
@@ -254,14 +254,53 @@ def parse_morbid_map(lines) -> Dict[str, List[str]]:
         tokens = line.split('\t')
         if not tokens or tokens == ['']:
             continue
-        m = p.match(tokens[0])
+        phenotype_label: str = tokens[0]
+        m = p.match(phenotype_label)
+        # TODO: How to get mapping key?
+        # m = p.match(tokens[1])  # ?
+        # todo: map to `phenotype_mapping_key_meanings`
+        # todo: return this info (more to do where func is called)
         if m:
             phenotype_mim_number = m.group(1)
         else:
             phenotype_mim_number = ''
-        gene_mim_number = tokens[2].strip()
-        cyto_location = tokens[3].strip()
+        gene_mim_number: int = tokens[2].strip()
+        gene_mim_number: str = str(gene_mim_number)
```

Comment:
is this good programming practice (changing the datatype of a variable)? I am sure a linter will scream :P

Reply:
The linter can kiss my butt, lol. I really do wish though that I felt less spread out and felt I had the time (or was disciplined enough) to have tests and lint / codestyle checkers passed on all my repos.

Yeah this is super ad hoc at this point. This is more a reminder to me that I would like to primarily treat this as `int` in the code, but right now the codebase is set up to read these MIM numbers as strings. I suppose adding another `todo` to the code would work; I tend to have them littered all over the place.